### DISCUSSION TITLE
Use carla with nDisplay plugin from UE4

### DISCUSSION THREAD

**Discussion by JDIRK3** [carla-simulator/carla#7751] [Q&A]
Hi,

does anybody successfully use carla with the nDisplay plugin from UE4?

I'm not sure whether this can work. I know that UE4.26 supports nDisplay and carla is based on 4.26, so there might be a chance.

**Comment by Junjie-0809** [comment]
Hi,

Are you successfully use carla with the nDisplay plugin from UE4?

I also wanted to try implementing UE's ndisplay feature with Carla recently.

I would appreciate if you Let me know  your progress.Thank you!

**Reply by JDIRK3** [reply]
Hi,

I never got it running. But I'm not quite sure, whether the problem was on my side or ndisplay.

Are you running it succesful?

**Comment by Junjie-0809** [comment]
Hi，


I tried but failed, so I plan to develop a plugin for integrated Carla and UE5 simulation later, enabling UE5 to handle ndisplay operation independently.




1569937727
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "carla-simulator/carla"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年6月2日(星期一) 下午5:40
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [carla-simulator/carla] Use carla with nDisplay plugin from UE4 (Discussion #7751)





 
Hi,
 
I never got it running. But I'm not quite sure, whether the problem was on my side or ndisplay.
 
Are you running it succesful?
